### PR TITLE
Added message for people who don't know where to right click with the floppy

### PR DESCRIPTION
**Pull request by skyem123**
I called it "idiot" because I am an idiot because I didn't get this.

**Blank disk**
Unused disk has notice
**Used disk**
Used disk has no notice

### CODE REVIEW

**Comment by skyem123** [comment]
I just realised that I forgot to localise the other stuff in the item... oops!

**Comment by skyem123** [comment]
Should be good now.

File: src/main/java/vic/mod/integratedcircuits/item/ItemFloppyDisk.java [hunk 1]
```diff
@@ -24,9 +25,14 @@ public void addInformation(ItemStack stack, EntityPlayer player, List itemInform
...
 			Integer size = comp.getInteger("size");
-			itemInformation.add(ChatFormatting.GRAY + "Name: " + ChatFormatting.WHITE + comp.getCompoundTag("properties").getString("name"));
-			itemInformation.add(ChatFormatting.GRAY + "Size: " + ChatFormatting.WHITE + size + "x" + size);
-			itemInformation.add(ChatFormatting.GRAY + "Author: " + ChatFormatting.WHITE + comp.getCompoundTag("properties").getString("author"));
+			itemInformation.add(ChatFormatting.GRAY + StatCollector.translateToLocalFormatted(getUnlocalizedName() + ".tooltip.name") + " " + ChatFormatting.WHITE + comp.getCompoundTag("properties").getString("name"));
+			itemInformation.add(ChatFormatting.GRAY + StatCollector.translateToLocalFormatted(getUnlocalizedName() + ".tooltip.size") + " " + ChatFormatting.WHITE + size + "x" + size);
+			itemInformation.add(ChatFormatting.GRAY + StatCollector.translateToLocalFormatted(getUnlocalizedName() + ".tooltip.author") + " " + ChatFormatting.WHITE + comp.getCompoundTag("properties").getString("author"));
+			/*itemInformation.add(ChatFormatting.WHITE + "" + ChatFormatting.ITALIC + StatCollector.translateToLocalFormatted(getUnlocalizedName() + ".tooltip.idiot.line1"));
```
Comment (Victorious3) [inline src/main/java/vic/mod/integratedcircuits/item/ItemFloppyDisk.java]: Ehhh....

Comment (skyem123) [inline src/main/java/vic/mod/integratedcircuits/item/ItemFloppyDisk.java]: What's wrong?

Comment (Victorious3) [inline src/main/java/vic/mod/integratedcircuits/item/ItemFloppyDisk.java]: There are comments! Why? Take them out!

**Comment by skyem123** [comment]
Better?

**Comment by Victorious3** [comment]
Yes... but do me a favor and try if using %n works, there is no point in making the tooltip two separate lines if the translation can decide where to put the line breaks.

**Comment by skyem123** [comment]
Sorry, I thought that what you thought about it. :/
Well, I'll fix it when I get home. I'm at school at the current time.

**Comment by Victorious3** [comment]
Well, that might be what I'm thinking but in my projects I try to stay serious ^^

**Comment by skyem123** [comment]
There we go!

**Pull request merged**